src/main/groovy/de/dkfz/execution/jobs/cluster/sge/SGEJobManager.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Where are the Direct Execution classes? Please extract the whol job.xyz.... packages! With all job managers